Library/Formula/subversion.rb

```diff
@@ -70,7 +74,11 @@ class Subversion < Formula
   end
 
   def install
-    serf_prefix = libexec+"serf"
+    # Fixes: "libtool: error: error: relink 'libsvn_ra_serf-1.la'
+    # with the above command before installing it"
+    # Please see https://github.com/Homebrew/linuxbrew/issues/408 for details
+    ENV.deparallelize
+    serf_prefix = OS.mac? ? libexec+"serf" : prefix
```

Comment:
Please include a comment above this line explaining why this fix is necessary. Simply quoting the error message is good enough.